src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatRequest {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
+    public boolean newMethod(String st1, String st2) {
+        if (st1 == st2) {
+            return true;
+        }
```

Comment:
Comparing strings using '==' operator checks for reference equality, not value equality. It is recommended to use '.equals()' method for comparing string values to ensure correct behavior.